mbq/__init__.py

```diff
@@ -1,2 +1,4 @@
 # https://packaging.python.org/namespace_packages/#pkgutil-style-namespace-packages
+import typing
+__path__: typing.Iterable[str]
```

Comment:
what made you want to type this?

Reply:
mypy

Reply:
sorry if this is a dumb question, just don't understand why you're typing this line in particular out of all of mbq.client. just proof of concepting mypy here?
 something else?